components/LinkMoreRoutes.vue

```diff
@@ -0,0 +1,31 @@
+<script setup>
+const props = defineProps({
+  name: {
+    type: String,
+    required: true
+  },
+  href: {
+    type: String,
+    required: true
+  },
+  rel: {
+    type: String,
+    default: 'noopener noreferrer'
```

Comment:
@PiotrDrechsler nie jestem do końca przekonana czy "noreferrer" powinien być w tym przypadku ustawiany domyślnie. Te linki będą prowadzić do wewnętrznych stron z trasami, a jeśli kiedyś będziemy podłączać jakąś analitykę do śledzenia ruchu na stronie, to te linki z "noreferrer" nie będą ujęte w takim przepływie.

Reply:
Racja. Wykasowane. Zostawiam sam "noopener".